No dataset for that day and that AS due to connectivity issues on the PlanetLab node.
